src/sentry/api/endpoints/relay/health_check.py

```diff
@@ -14,8 +14,8 @@ class RelayHealthCheck(Endpoint):
     Relay doesn't need to care if it connects to another Relay or directly to sentry.
     """
 
-    authentication_classes = []
-    permission_classes = []
+    authentication_classes = []  # type: ignore
```

Comment:
Why are these ignored?

Reply:
These variables override their parents so they're also inherit types. I wanted to wait for `src/sentry/api/base.py` to get added to mypy but I think the right thing to is just explicitly type now instead of ignoring.